plutus-contract/src/Plutus/Contract/Test/ContractModel/Internal.hs

```diff
@@ -1513,7 +1513,7 @@ propRunActions_ actions =
 -- to write `ContractModel`s that keep track of balances.
 defaultCheckOptionsContractModel :: CheckOptions
 defaultCheckOptionsContractModel =
-  defaultCheckOptions & emulatorConfig . initialChainState .~ (Left . Map.fromList $ zip knownWallets (repeat (Ada.lovelaceValueOf 100_000_000_000_000_000)))
+  defaultCheckOptions & increaseTransactionLimits & emulatorConfig . initialChainState .~ (Left . Map.fromList $ zip knownWallets (repeat (Ada.lovelaceValueOf 100_000_000_000_000_000)))
```

Comment:
I don't think we want this as a default value. We should instead use `increaseTransactionLimits` on a case-by-case basis.